community/cypher/cypher-planner-3.4/src/test/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/logical/idp/IDPQueryGraphSolverTest.scala

```diff
@@ -44,10 +44,10 @@ class IDPQueryGraphSolverTest extends CypherFunSuite with LogicalPlanningTestSup
     new given {
       queryGraphSolver = createQueryGraphSolver(monitor = monitor, solverConfig = EmptySolverConfig())
       qg = QueryGraph(patternNodes = Set("a"))
-    }.withLogicalPlanningContext { (cfg, ctx) =>
-      val plan = queryGraphSolver.plan(cfg.qg, ctx)
+    }.withLogicalPlanningContext { (cfg, ctx, solveds, cardinalities) =>
```

Comment:
This whole file looks like it might work with Fake Attributes.

Reply:
Also nope, sadly.